composer.json

```diff
@@ -24,12 +24,14 @@
         "friendsofsymfony/jsrouting-bundle": "^2.4",
         "geoip2/geoip2": "~2.4.2",
         "greenlion/php-sql-parser": "^4.1",
+        "guzzlehttp/cache-subscriber": "^0.2.0",
```

Comment:
is this really mandator for this pull request?

Reply:
Yes because it should have already been there As explained in another comment the circuit breaker was integrated in the core without this dependency No error occurred thanks to some modules integrating it in their own dependencies but if you uninstall those modules then the core won't have it any more

Reply:
If it's because of the circuit-breaker, the circuit-breaker needs to be updated, not the Core :thinking:

Reply:
No because in the circuit breaker this dependency is only a suggestion https://github.com/PrestaShop/circuit-breaker/blob/ba30dc3259fb166a0530b6d510ad1bf2e4629ba5/composer.json#L33

You can use it without any cache

Reply:
Oh, great!